Trying simpler layout and styles

eigilhs opened the pull request:
See http://cleanup.pr.dev.xal.no/

skavhaug commented:
How do we go about reviewing this changeset? Quite a lot have been done, and its a bit unclear which changes to keep and which to work on further. Also, pretty easy to get stuck in bike sheding here :)

skavhaug commented:
General comment is that I like a lot of the changes, but I think there are too many different fonts. It looks a bit messy, e.g. the services page.

eigilhs commented:
Most of the changes in `/layout` is just making things behave the way we originally intended. E.g. I have added a `<section>` around some of the content to fix some places where we got double margins between paragraphs. I have also deleted some unused CSS.

The main changes are the background color, fonts, and margins.
